patches/@bitdrift+react-native+0.4.0.patch

```diff
@@ -0,0 +1,72 @@
+diff --git a/node_modules/@bitdrift/react-native/fabric/NativeBdReactNative.ts b/node_modules/@bitdrift/react-native/fabric/NativeBdReactNative.ts
```

Comment:
FWIW I heard from bitdrift folks that they released an update that should make this unnecessary